addons/account/wizard/account_resequence.py

```diff
@@ -137,6 +137,9 @@ def resequence(self):
                 raise UserError(_('You can not reorder sequence by date when the journal is locked with a hash.'))
         self.env['account.move'].browse(int(k) for k in new_values.keys()).name = False
         for move_id in self.move_ids:
+            if move_id.company_id.fiscalyear_lock_date and move_id.date and move_id.date <= move_id.company_id.fiscalyear_lock_date:
+                raise UserError(_('You cannot resequence an invoice in a locked period (%s).',
+                                  format_date(self.env, move_id.company_id.fiscalyear_lock_date)))
```

Comment:
You should use `_check_fiscalyear_lock_date` instead

Reply:
> But really, I think that the only thing you need to do is add name in fiscal_fnames in _get_lock_date_protected_fields
`_get_lock_date_protected_fields` is defined for lines (which have no `name`, not for moves. So I'll go with `_check_fiscalyear_lock_date` :)

Reply:
It wasn't a "or" kind of situation, `_check_fiscalyear_lock_date` doesn't do anything since the `name` is not in the protected fields list (did you test?)

Do it with `move_name` then, it is the related (stored) field.

Reply:
`_check_fiscalyear_lock_date` does do the check, at least in 14.0: https://github.com/odoo/odoo/blob/7fbb337e31f0e7e1a6714d8dbf73facb00854dda/addons/account/models/account_move.py#L1859-L1868 so yeah the error is displayed as expected.
The resequence works on the `account.move`, not on its lines, so the `write` that checks if the lock date is violated is never performed (so `move_name` adding here https://github.com/odoo/odoo/blob/7fbb337e31f0e7e1a6714d8dbf73facb00854dda/addons/account/models/account_move.py#L4152 won't do the trick I think)